app/src/main/java/com/example/campusguide/directions/ChooseDirectionOptionsDialogFragment.kt

```diff
@@ -20,7 +20,7 @@ import com.google.android.gms.location.LocationServices
 import com.google.android.gms.tasks.OnCompleteListener
 
 
-class ChooseDirectionOptions constructor(private val route: Route?) : DialogFragment() {
+class ChooseDirectionOptionsDialogFragment constructor(private val route: Route?) : DialogFragment() {
```

Comment:
Is it necessary to add "DialogFragment" in the class name if it already extends DialogFragment()? I'm not sure what the best practice is in this case

Reply:
IMO it should at least have Dialog, and i'd be okay with DialogFragment

Why? ChooseDirectionOptions sounds like a data transfer object containing the options available when choosing directions. That is not what this class does at all: it's a UI component, and having DialogFragment, or at least Dialog, conveys that information well.